<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFilmRequisition</class>
 <widget class="QDialog" name="NewFilmRequisition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Film Requisition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" rowspan="2" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Artwork Size (mm)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="spinArtworkLength"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinArtworkWidth"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QComboBox" name="cboSalesRep"/>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="spinNoOfColors"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="cboCustomer"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>No. Of Colors:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="cboProduct"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sales Rep:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Product:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Customer:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Sterio Size (in):</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="spinSterioLength"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinSterioWidth"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Chargeable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cmdOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
